src/_images.css

```diff
@@ -5,7 +5,19 @@
 
 */
 
-/* Responsive images! */
+img { display: block; max-width: 100%; }
```

Comment:
Agreed. I appreciate the principle of responsive by default, but this seems like overreach. maybe it could be opt in? maybe `.responsive-image` to match the ones below??